svc-managers/mgrmodel/manager.go

```diff
@@ -160,19 +160,19 @@ func GetManagerByURL(url string) (string, *errors.Error) {
 	return manager, nil
 }
 
-// UpdateManagersData will modify the current details to given changes
-func UpdateManagersData(key string, managerData map[string]interface{}) error {
+// UpdateData will modify the current details to given changes
+func UpdateData(key string, managerData map[string]interface{}, table string) error {
```

Comment:
Why are we changing these functions? Was there an architecture discussion? BB why isnt team catching these in review ?

Reply:
managerData looks familiar ?

Reply:
As above what other data are we updating in manager module?

Reply:
Earlier this function was using a hardcoded value for update, I have reused the function by passing a parameter

Reply:
can i send a non managerData to this func ?

Reply:
Now this function can update data based on the key passed, earlier it was updating only manager key data

Reply:
Then lets update paramname "managerData"

Reply:
Already the param name is changed, kindly check